lib/internal/Magento/Framework/App/Response/HttpInterface.php

```diff
@@ -16,4 +16,49 @@
      * @return void
      */
     public function setHttpResponseCode($code);
+
```

Comment:
Also the method was used by many clients anyway, so any custom implementation would have not worked within the context of Magento 2 without this method anyway.